src/content/app/tools/blast/state/blast-api/blastApiSlice.ts

```diff
@@ -99,6 +99,37 @@ const blastApiSlice = restApiSlice.injectEndpoints({
         };
       }
     }),
+    fetchAllBlastJobs: builder.query<BlastJobResultResponse[], string[]>({
```

Comment:
Aren't we fetching the result of all the jobs here instead of the jobs itself? Should it be `fetchAllBlastJobsResult`?

Reply:
It's getting too long. And the hook will become `useFetchAllBlastJobResultsQuery` 🤢 I'd rather keep the name as is.